client/my-sites/activity/activity-log-banner/progress-banner.jsx

```diff
@@ -67,15 +67,15 @@ function ProgressBanner( {
 						? translate( 'The cloning process will start in a moment.' )
 						: translate( 'Away we go! Your site is being cloned.' );
 			} else {
-				title = translate( 'Currently rewinding your site' );
+				title = translate( 'Currently restoring your site' );
 				description = translate(
-					"We're rewinding your site back to %(dateTime)s. You'll be notified once it's complete.",
+					"We're restoring your site back to %(dateTime)s. You'll be notified once it's complete.",
 					{ args: { dateTime } }
 				);
 				statusMessage =
 					'queued' === status
-						? translate( 'Your rewind will start in a moment.' )
-						: translate( 'Away we go! Your site is being rewound.' );
+						? translate( 'Your restore will start in a moment.' )
+						: translate( 'Away we go! Your site is being restored.' );
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.